### DISCUSSION TITLE
A bit of press

### DISCUSSION THREAD

**Discussion by Xylopyrographer** [Xylopyrographer/STAC#39] [General]
@mqshaw 
Hi Mark:

Hope your summer is going well.

We got a bit of a shout out from the techminisry.blog folks in their release notes for v2 of Tally Arbiter:
[https://techministry.blog/2021/07/29/tally-arbiter-2-0-now-available/](https://techministry.blog/2021/07/29/tally-arbiter-2-0-now-available/)

**Comment by mqshaw** [comment]
That’s great, thanks for the heads up.

Mark


From: Xylopyrographer ***@***.***>
Sent: Thursday, July 29, 2021 11:22 AM
To: Xylopyrographer/STAC ***@***.***>
Cc: Mark Shaw ***@***.***>; Mention ***@***.***>
Subject: [Xylopyrographer/STAC] A bit of press (#39)


@mqshaw<https://github.com/mqshaw>
Hi Mark:

Hope your summer is going well.

We got a bit of a shout out from the techminisry.blog folks in their release notes for v2 of Tally Arbiter:
https://techministry.blog/2021/07/29/tally-arbiter-2-0-now-available/
